src/applications/ezr/config/form.js

```diff
@@ -420,9 +421,12 @@ const formConfig = {
           uiSchema: {},
           schema: VIEW_FIELD_SCHEMA,
         },
-        financialInformationIntroduction: {
-          ...FinancialInformationPages.financialInformationIntroduction,
+        financialInformationOverview: {
```

Comment:
Since we updated the introPage entry for this page. We don't need this config any longer. What I'd like to do with this is spread `FinancialInformationPages` there on L423, then get rid of `financialInformationOverview` we have in here along with `financialInformationSummary` on L427, `veteranAnnualIncomeV2` on L439, `spouseAnnualIncomeV2` on L451 and `deductibleExpensesV2` on L463.

Reply:
Sounds good. Done.